ets-demo/etsdemo/tests/test_info.py

```diff
@@ -0,0 +1,44 @@
+# (C) Copyright 2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" Test the EAM app metadata.
+"""
+
+import os
+import subprocess
+import unittest
+
+try:
+    import eam   # noqa: #F401
+except ImportError:
+    EAM_EXISTS = False
```

Comment:
i don't think `eam` can be an optional dependency so i'm not sure why we want to have a try/import here.

Reply:
Why can't it be an optional dependency?
It is currently only a test dependency. Without eam, the application can still run.

Reply:
>  It is currently only a test dependency. Without eam, the application can still run.

Ohh. Ha. Fair enough. Sorry.